src/preact/base-element.js

```diff
@@ -419,13 +419,22 @@ export class PreactBaseElement extends AMP.BaseElement {
   }
 
   /** @override */
-  attachedCallback() {
+  mountCallback() {
     discover(this.element);
+    const Ctor = this.constructor;
+    if (Ctor['loadable'] && this.getProp('loading') != Loading.AUTO) {
```

Comment:
This overrides, is that intentional? My feeling is that it should only set when `getProp` is empty?

Reply:
The `loading` is not currently exposed to the public API so it's only used for internal needs here and we don't need to worry about overriding it. The only reason we do here `getProp(loading) != AUTO` is because the `mutateProps()` is a bit inefficient - even if a property hasn't changed, it'd still force rerendering and here we can avoid it.